hbase-assembly/pom.xml

```diff
@@ -318,5 +318,18 @@
       <artifactId>jaxws-ri</artifactId>
       <type>pom</type>
     </dependency>
+    <!--
+      Include the log framework here.
+      For other sub modules, we only declare slf4j-api as a compile dependency,
+      so here we must pull in the real logging framework to actually log to log4j.
+    -->
```

Comment:
If the tarball is built by running assembly, we will include the slf4j and log4j jars, and also the log4j.properties under conf directory, so it will produce logs.

Reply:
I'm asking about running an hbase instance from built checkout, not from an assembly. Currently it works. It looks like it will no longer work if this patch goes in?

Reply:
We could do this? I'v never tried this before. How does the start-hbase.sh script find all the dependencies? We do not have a lib directory in the checkout directory...